packages/remix/src/executors/build/build.impl.ts

```diff
@@ -1,50 +1,87 @@
-import { logger, writeJsonFile, type ExecutorContext } from '@nx/devkit';
+import {
+  logger,
+  readJsonFile,
+  writeJsonFile,
+  type ExecutorContext,
+} from '@nx/devkit';
 import { createLockFile, createPackageJson, getLockFileName } from '@nx/js';
 import { directoryExists } from '@nx/workspace/src/utilities/fileutils';
-import { cli as remixCli } from '@remix-run/dev';
+import { fork } from 'child_process';
 import { copySync, mkdir, writeFileSync } from 'fs-extra';
 import { type PackageJson } from 'nx/src/utils/package-json';
 import { join } from 'path';
 import { type RemixBuildSchema } from './schema';
 
+function buildRemixBuildArgs(options: RemixBuildSchema) {
+  const args = ['build'];
+
+  if (options.sourcemap) {
+    args.push(`--sourcemap`);
+  }
+
+  return args;
+}
+
+async function runBuild(
+  options: RemixBuildSchema,
+  context: ExecutorContext
+): Promise<void> {
+  const projectRoot = context.projectGraph.nodes[context.projectName].data.root;
+  return new Promise<void>((resolve, reject) => {
+    const remixBin = require.resolve('@remix-run/dev/dist/cli');
+    const args = buildRemixBuildArgs(options);
+    const p = fork(remixBin, args, {
+      cwd: join(context.root, projectRoot),
+      stdio: 'inherit',
```

Comment:
should you pass `env` variables here?

Reply:
It's chained by default if you dont' pass it. :)